### PR TITLE
feat(AWS SNS): Recognise `filterPolicyScope` event definition

### PR DESCRIPTION
**Pull request by jasonrowsell**
Closes: #11641

### CODE REVIEW

File: lib/plugins/aws/package/compile/events/sns.js [hunk 1]
```diff
@@ -23,6 +23,7 @@ class AwsCompileSNSEvents {
             topicName: { type: 'string', maxLength: 256, pattern: '^[\\w-]+$' },
             displayName: { type: 'string', minLength: 1 },
             filterPolicy: { type: 'object' },
+            filterPolicyScope: { enum: ['MessageBody', 'MessageAttributes'] },
```
Comment (medikoo) [inline lib/plugins/aws/package/compile/events/sns.js:26]: Let's maybe simplify it to `{ type: "string" }`, so value is validated on AWS side and we do not have dependency on it in the Framework

**Review by medikoo — COMMENTED** [review]
Thank you!

File: lib/plugins/aws/package/compile/events/sns.js (hunk 1 above)
Comment (jasonrowsell) [reply lib/plugins/aws/package/compile/events/sns.js]: Updated to string :)

**Review by medikoo — APPROVED** [review]
Thank you!

**Pull request merged**